### DISCUSSION TITLE
I have a question of "Github Pages"

### DISCUSSION THREAD

**Discussion by shidijun** [community/community#144738] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

(My English is poor.Pardon please.)
My "Github Page" is displaying the "RENDME.md".Can I let it display what I want html file?

<a href="https://shidijun.github.io/blog">The website</a>

我的“Github页面”显示“RENDME.md”。我可以让它显示我想要的html文件吗？

<a href="https://shidijun.github.io/blog">所述网站</a>

**Comment by Anonav0** [comment, marked as answer]
If you are trying to use github pages. Then you need to have `index.html` page in your repo. I guess you are having `something.html`. try to change that `something.html` to `index.html'. It'll work.

**Reply by shidijun** [reply]
Indeed, that's right. Just now, a kind-hearted person helped me, so I successfully made the correct changes. Anyway, thank you~

**Reply by Anonav0** [reply]
No problem buddy. Please mark this discussion as answered 🙌.

**Reply by shidijun** [reply]
Ok.

**Comment by Anonav0** [comment]
It seems like your pages are working fine now.
